idb/postgres/postgres_migrations.go

```diff
@@ -821,6 +821,28 @@ func m7RewardsAndDatesPart2UpdateAccounts(db *IndexerDb, accountData []addressAc
 	return nil
 }
 
+func warnUser(db *IndexerDb, maxRound uint32) error {
+	query := "SELECT COUNT(*) FROM account WHERE (created_at IS NULL) OR (created_at <= $1)"
+	row := db.db.QueryRow(query, maxRound)
+
+	var count uint64
+	err := row.Scan(&count)
+	if err != nil {
+		return fmt.Errorf("m7: unable to query the number of rows: %v", err)
+	}
+
+	if count > 10000000/3*4 {
```

Comment:
This is a magic number. Consider putting it in a constant.

Reply:
Where should I put it?

Reply:
I think defining it above the condition is fine along with a comment is fine, no need to spread this logic around:
```
// This many accounts needs about 4GB of memory
maxAccountsWithoutWarning := 10000000/3*4
```